src/pages/country-data.js

```diff
@@ -0,0 +1,87 @@
+export const UNSC = [
+  'Belgium ',
+  'Dominican Republic ',
+  'Estonia',
+  'Germany ',
+  'Indonesia ',
+  'Niger',
+  'Saint Vincent and the Grenadines',
+  'South Africa ',
+  'Tunisia',
+  'Viet Nam',
+]
+
+export const UNHRC = []
+
+export const DISEC = []
+
+export const LokSabha = []
+
+export const CSW = [
+  'Albania',
+  'Estonia',
+  'Namibia',
+  'Ghana',
+  'Niger',
+  'Bangladesh',
+  'Guatemala',
+  'Nigeria',
+  'Belgium',
+  'Guyana',
+  'Norway',
+  'Bosnia and Herzegovina',
+  'India',
+  'Peru',
+  'Brazil',
+  'Iraq',
+  'Qatar',
+  'Canada',
+  'Ireland',
+  'Republic of Korea',
+  'Chile',
+  'Israel',
+  'Russian Federation',
+  'China',
+  'Kazakhstan',
+  'Spain',
+  'Colombia',
+  'Kenya',
+  'Tajikistan',
+  'Congo',
+  'Kuwait',
+  'Trinidad and Tobago',
+  'Egypt',
+  'Liberia',
+  'Tunisia',
+  'El Salvador',
+  'Liechtenstein',
+  'United Kingdom',
+  'Equatorial Guinea',
+  'Malawi',
+  'United Republic of Tanzania',
+  'Eritrea',
+  'Mongolia',
+  'Uruguay',
+]
+export const IMF = []
+
+export const CSTD = []
+
+export const NSG = []
+
+export const SPECPOL = []
+
+export const WHA = []
+
+export const Committe = {
+  0: UNSC,
+  1: UNHRC,
+  2: DISEC,
+  3: LokSabha,
+  4: CSW,
+  5: IMF,
+  6: CSTD,
+  7: NSG,
+  8: SPECPOL,
+  9: WHA,
+}
```

Comment:
@ksvkabra The pages folder is only for keeping pages. Don't put any data in this folder. It will work fine in development but will fail in production. 
Create a seperate `data` folder in the `src` directory and put it in there.